README.md

```diff
@@ -1,2 +1,15 @@
 # ernm
 Estimation of fully/partially observed Exponential-Family Random     Network Models (ERNM). ERNMs are a generalization of ERGM (see the ergm     package) and Gibbs Fields, encompassing both as special cases
+
+
+## Adding Statistics
```

Comment:
Note: The ideal way for a third party to add statistics is for them to create their own empty package, have it link to ernm, write just their stats into the package, and then register the stat on load by calling `register*Statistic`.

Reply:
I know this is the intended way - and makes sense for "specialized" stats. I was thinking to encourage users to make their stats available to all ERNM users? I'll add some words to this effect